packaging/nnstreamer.spec

```diff
@@ -60,6 +60,15 @@
 %define		edgetpu_support 0
 %endif
 
+# tizen 6.0 (or less) backward-compatibility check
+%if ( 0%{tizen_version_major} == 6 && 0%{tizen_version_minor} < 5 ) || 0%{tizen_version_major} < 6
```

Comment:
```0%{tizen...``` --> ```0%{?tizen...```

Reply:
It seems both work. But It's better to use the formal one, thanks.

Reply:
It's not just "formal". It behaves differently: https://backreference.org/2011/09/17/some-tips-on-rpm-conditional-macros/

Reply:
Oh.. thanks for the info. Actually, I just used the similar conditions in the existing spec file.
https://github.com/nnstreamer/nnstreamer/blob/5ce00070d8423d5f90e88967abb7078f7a7a4aa9/packaging/nnstreamer.spec#L44-L51

Anyway, in this case, `0%{tizen...` and `0%{?tizen...` behaves similarly because we didn't set the default values.
```
%{?mymacro:%{mymacro}} can also be written just as %{?mymacro}
```